tasks/sync-granule/schemas/config.json

```diff
@@ -87,6 +87,7 @@
         "name": { "type": "string" },
         "path": { "type": "string" }
       }
-    }
+    },
+    "duplicateHandling": { "type": "string" }
```

Comment:
It would be good to list out the valid values that can be used here using `enum`.

https://ajv.js.org/keywords.html#keywords-for-all-types

Reply:
This should probably be part of a separate ticket since it is not clear what the enum should be.